sky/jobs/dashboard/dashboard.py

```diff
@@ -6,13 +6,18 @@
 rid of the SSH port-forwarding business (see cli.py's job_dashboard()
 comment).
 """
+import collections
 import datetime
+import os
 import pathlib
 
 import flask
+from flask import abort
+from flask import send_file
```

Comment:
styleguide nit: import modules only, not individual funcs/members. https://google.github.io/styleguide/pyguide.html#313-imports-formatting